llvm/lib/Transforms/Instrumentation/MemorySanitizer.cpp

```diff
@@ -3043,7 +3049,7 @@ struct MemorySanitizerVisitor : public InstVisitor<MemorySanitizerVisitor> {
     }
 
     if (I.doesNotAccessMemory())
-      if (maybeHandleSimpleNomemIntrinsic(I))
+      if (maybeHandleSimpleNomemIntrinsic(I, /* trailingFlags */ 0))
```

Comment:
I believe convention is `/*trailingFlags=*/`, please double check

Reply:
Thanks, fixed in https://github.com/llvm/llvm-project/pull/124421/commits/56c0267f91967d6db9f9e9f5f93ad63343464230